.automation.conf/tempest/tempest-ci-multinode.overrides.conf

```diff
@@ -10,7 +10,7 @@ v3_endpoint_type = publicURL
 [compute]
 min_compute_nodes = 2
 min_microversion = 2.1
-max_microversion = 2.88
+max_microversion = 2.93
```

Comment:
2.95 in antelope: https://docs.openstack.org/nova/latest/reference/api-microversion-history.html#maximum-in-2023-1-antelope-and-2023-2-bobcat

Reply:
Agree this needs to change, but this is just a merge from Zed PR